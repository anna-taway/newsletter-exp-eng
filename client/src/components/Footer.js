import React from 'react';
import { makeStyles } from '@material-ui/core/styles';
import Typography from '@material-ui/core/Typography';

const useStyles = makeStyles((theme) => ({
    icon: {
        marginRight: theme.spacing(2),
    },
    footer: {
        backgroundColor: theme.palette.background.paper,
        padding: theme.spacing(6),
    }
}));

const Footer = () => {
    const classes = useStyles();
    return (
        <React.Fragment>
            <footer className={classes.footer}>
            <Typography variant="h6" align="center" gutterBottom>
                A production with EE and EBQ
            </Typography>
            <Typography variant="subtitle1" align="center" color="textSecondary" component="p">
                Contact
            </Typography>
            </footer>
        </React.Fragment>
    )
}

export default Footer;