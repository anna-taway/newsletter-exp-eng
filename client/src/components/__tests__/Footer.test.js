import React from "react";
import Footer from "../Footer";
import { create } from "react-test-renderer";

describe("Footer component", () => {
  test("Footer component matches the snapshot", () => {
    const footer = create(<Footer />);
    expect(footer.toJSON()).toMatchSnapshot();
  });
});