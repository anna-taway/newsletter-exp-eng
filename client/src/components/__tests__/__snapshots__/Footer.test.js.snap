// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Footer component matches the snapshot 1`] = `
<footer
  className="makeStyles-footer-2"
>
  <h6
    className="MuiTypography-root MuiTypography-h6 MuiTypography-gutterBottom MuiTypography-alignCenter"
  >
    A production with EE and EBQ
  </h6>
  <p
    className="MuiTypography-root MuiTypography-subtitle1 MuiTypography-colorTextSecondary MuiTypography-alignCenter"
  >
    Contact
  </p>
</footer>
`;
